events/eleventh_street.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun 17 20:49:38 2019
+
+@author: peterreischer
+"""
+
+from datetime import datetime
+import sys
+import requests
+from bs4 import BeautifulSoup
+import logging
+import re
+import math
+
+logger = logging.getLogger(__name__)
+
+def soupify_event_page(url = 'https://bbardc.org/events/'):
+    try:
+        r = requests.get(url)
+    except Exception as e:
+        logger.critical(f'Exception making GET to {url}: {e}', exc_info = True)
+        return
+    content = r.content
+    soup = BeautifulSoup(content, 'html.parser')
+
+    return soup
+
+def soupify_event_website(event_website):
+    try:
+        r = requests.get(event_website)
+    except Exception as e:
+        logger.critical(f'Exception making GET to {event_website}: {e}', exc_info = True)
+        return
+    content = r.content
+    soup = BeautifulSoup(content, 'html.parser')
+
+    return soup
+
+
+
+def schematize_event_time(event_startTime):
+    '''
+    Converts a time string like 'May 30 @ 1:30 pm' to 24hr time like '13:30:00'
+    '''
+
+    try:
+        datetime_obj = datetime.strptime(event_startTime.split(' @ ',1)[1], "%I:%M %p")
+        schematized_event_time = datetime.strftime(datetime_obj, "%H:%M:%S")
+    except IndexError:
+        schematized_event_time = ''
+    except ValueError:
+        logger.warning(f'Exception schematizing this time: {event_startTime}', exc_info = True)
+        schematized_event_time = ''
+
+    return schematized_event_time
+
+def schematize_event_dates(event_id, start_date, end_date):
+    '''
+    Convert a date string like 'May 30 @ 1:30 PM' to a date like 2019-05-30.
+    Checks what year the event is taking place in.
+
+    '''
+    start_date_out = ''
+    end_date_out = ''
+    all_day_event = False
+    try:
+        start_date = datetime.strptime(start_date.split(' @ ', 1)[0], '%B %d')
+        end_date = datetime.strptime(end_date.split(' @ ', 1)[0], '%B %d')
+        id_date = datetime.strptime(event_id[5:], '%Y-%m-%d')
+        start_date = start_date.replace(year = id_date.year)
+        end_date = end_date.replace(year = id_date.year)
+        if start_date > end_date:
+            start_date = start_date.replace(year = id_date.year - 1)
+        if (end_date-start_date).days >= 1:
+            all_day_event = True
+        start_date_out = datetime.strftime(start_date, '%Y-%m-%d')
+        end_date_out = datetime.strftime(end_date, '%Y-%m-%d')
+    except IndexError:
+        pass
+    except ValueError:
+        logger.warning(f'Exception schematizing this date: {start_date}', exc_info = True)
```

Comment:
For these two lines, can you add the event's website to the logged message. That'll help with debugging.